src/uct/ib/ud/base/ud_ep.c

```diff
@@ -546,7 +545,8 @@ static ucs_status_t uct_ud_ep_create(uct_ud_iface_t *iface, int path_index,
     uct_ep_h new_ep_h;
     ucs_status_t status;
 
-    status = uct_ep_create(&params, &new_ep_h);
+    status = uct_iface_invoke_ops_func(&iface->super, uct_ud_iface_ops_t,
+                                       ep_create, &params, 0, &new_ep_h);
```

Comment:
this change, and passing flags, seems not needed: we could check params and set CONNECT_TO_EP flag according to params content, after creating the ep